dkc/core/models/file.py

```diff
@@ -34,7 +34,7 @@ class Meta:
         ],
     )
     description = models.TextField(max_length=3000, blank=True)
-    size = models.PositiveBigIntegerField(editable=False)
+    size = models.PositiveBigIntegerField()
     content_type = models.CharField(max_length=255, default='application/octet-stream')
     blob = S3FileField()
```

Comment:
```suggestion
    blob = S3FileField(blank=True)
```

This is only enforced for forms (including Django admin), but technically this field is now optional.